features/metal/info.yaml

```diff
@@ -3,6 +3,7 @@ type: platform
 features:
   include: 
     - server
+    - _secureboot
```

Comment:
as for cloud secureboot probably also shouldn't be included by default for metal either

Reply:
Removed it as well